voyager-spring-boot-autoconfigure/build.gradle

```
@@ -21,9 +21,9 @@ dependencies{
     
     compile "org.springframework.boot:spring-boot-autoconfigure:$LIB_SPRING_BOOT_VER"
     compile "org.apache.commons:commons-text:1.1"
-    compileOnly "org.springframework.boot:spring-boot-starter-web:$LIB_SPRING_BOOT_VER"
+    compile "org.springframework.boot:spring-boot-starter-web:$LIB_SPRING_BOOT_VER"
```

Would make these two `compileOnly` to force the user to provide either one of the dependencies instead of pulling them both in.
```suggestion
    compileOnly "org.springframework.boot:spring-boot-starter-web:$LIB_SPRING_BOOT_VER"
    compileOnly "org.springframework.boot:spring-boot-starter-webflux:$LIB_SPRING_BOOT_VER"
```